### DISCUSSION TITLE
Config not saving after restart

### DISCUSSION THREAD

**Discussion by Maqqnus** [Rem0o/FanControl.Releases#1501] [Q&A]
Does anyone know why my settings aren't saving when I restart the pc? Meaning; I have to re-apply my fan curves to the controls on every startup, and changes I make to the curves don't get saved (like changing temp source).

This might be a dumb question, but I can't figure this out and I'd appreciate some help :)

**Comment by Rem0o** [comment]
Did you ever save your config with a dedicated name ( save as ) ?

**Reply by Maqqnus** [reply]
No I didn't, but that seems to have done the trick. I'm stupid. Thanks!
